content/03.methods.md

```diff
@@ -35,22 +35,43 @@ These articles were the first 200 English language "Journal" articles returned b
 Character deletion was assigned a zero cost because we would like exact substring matches.
 For example, the calculated cost, including a cost for character deletion, between John and John Steinberg is 10; without character deletion, it is 0.
 Compared with the distance between John and Jane Doe, with character deletion cost, it is 7; without it is 2.
-Furthermore, the identification of simultaneously cited and quoted speakers used this same name matching strategy.
- 
+If we are still unable to find a full name, or if coreNLP cannot identify a speaker at all, we also look if coreNLP identified a gendered pronoun to the quote.
+If so, we assign the gender of the speaker to the gender of the pronoun.
+We ignore all quotes with no name or partial names and no associated pronouns.
+
+
+#### Name Formatting for Gender Prediction of Authors
+Only articles with more than one author were considered.
+Similarly for the name formatting for quotes, we need to extract the first name of the authors.
+Names were cast to lowercase and processed using the R package humaniformat [@https://cran.r-project.org/web/packages/humaniformat/index.html].
+humaniformat identifies if names are reversed (Lastname, Firstname), as well as identifies middle names.
+This processing was not done in the quotes prediction because we assume the names written in an article are not reversed or abbreviated.
+Since many last or first authorships may be non-names, we additionally filtered out any identified names if they partially or fully match any of the following terms: "consortium", "group", "initiative", "team", "collab", "committee", "center", "program", "author", or "institute".
+Furthermore, since many articles only contain first name initials (for example, "N. Davidson"), we remove any names less than four letters (length includes punctuation) and containing a  "." or "-", then strip out all periods from the first name.
+This ensures that hyphenated names are not changed, e.g. Julia-Louise remains unchanged, but removes hyphenated initials, e.g. J-L.
+Finally, we only consider any remaining first names of more than two characters.
+This is to eliminate first and middle jointly-initialized names.
+For example, "NR Davidson" would be reduced to "Davidson" and then eliminated due to the lack of a first name.
+
+
+#### Name Formatting for Name Origin Prediction
+In contrast to the gender prediction, we require the entire name in all steps of name origin prediction.
+For names identified in the _Nature_ news articles, we use the same process as described for the gender prediction; we again try to identify the full name.
+For author names, we process the names as previously described for the gender prediction of authors.
+For all names, we only consider them in our analyses if they consist of two distinct parts separated by a space.
+Additionally, if a full name is less than three characters, it is not considered.
```

Comment:
```suggestion
Additionally, if a full name is less than three characters, we were unable to consider it as the prediction model that we apply uses 3-mers.
```
I think this is a technical constraint that would make it impossible to apply without retraining - so we can note that as a reason.